docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md

```diff
@@ -42,18 +42,18 @@ Watch the following micro lesson to learn about Cloud Infrastructure Security fo
 ## Log types
 
 Cloud Infrastructure Security for AWS utilizes the following log types:
-* [Amazon CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html) 
+* [Amazon CloudTrail](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-getting-started.html)
 * [Amazon GuardDuty](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_findings.html)
 * [AWS Network Firewall](https://docs.aws.amazon.com/network-firewall/latest/developerguide/what-is-aws-network-firewall.html)
 * [AWS Security Hub](https://docs.aws.amazon.com/securityhub/latest/userguide/what-is-securityhub.html)
-* [AWS Web Application Firewall (WAF)](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html) 
+* [AWS Web Application Firewall (WAF)](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html)
 
-## Sample log message
+## Sample log messages
```

Comment:
I know you're just trying to standardize, but why would you change "Sample log message" to "Sample log messages" when there is only one log message shown?

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/3558#discussion_r1464183159

Reply:
Okay, @kimsauce. That makes sense.